pom.xml

```diff
@@ -0,0 +1,248 @@
+		<commons-lang3.version>3.4</commons-lang3.version>
+		<jackson-mapper-asl.version>1.9.13</jackson-mapper-asl.version>
+		<dom4j.version>1.6.1</dom4j.version>
+		<axis.version>1.4</axis.version>
+		<jaxen.version>1.1.6</jaxen.version>
+
+		<testng.version>6.8.21</testng.version>
+		<mockito-all.version>1.10.19</mockito-all.version>
+	</properties>
+
+	<!-- <licenses> <license> <name>MIT License</name> <url>http://opensource.org/licenses/MIT</url> <distribution>repo</distribution> </license> </licenses> -->
+
+	<scm>
+		<connection>scm:git:git@github.com:adlogix/appnexus-oas-java-client.git</connection>
+		<developerConnection>scm:git:git@github.com:adlogix/appnexus-oas-java-client.git</developerConnection>
+		<url>git@github.com:adlogix/appnexus-oas-java-client.git</url>
+	</scm>
+
+	<distributionManagement>
+		<snapshotRepository>
+			<id>ossrh</id>
+			<url>https://oss.sonatype.org/content/repositories/snapshots</url>
+		</snapshotRepository>
+		<repository>
+			<id>ossrh</id>
+			<url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+		</repository>
+	</distributionManagement>
+
+	<build>
+		<plugins>
+
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<version>${mvn.compiler.version}</version>
+				<configuration>
+					<compilerVersion>${java.version}</compilerVersion>
+					<source>${java.version}</source>
+					<target>${java.version}</target>
+				</configuration>
+			</plugin>
+
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-failsafe-plugin</artifactId>
+				<version>2.18.1</version>
+				<configuration>
+					<systemPropertyVariables>
+						<credentialsFile>${oas.credentials.file}</credentialsFile>
+					</systemPropertyVariables>
+				</configuration>
+				<executions>
+					<execution>
+						<id>integration-test</id>
+						<goals>
+							<goal>integration-test</goal>
+						</goals>
+					</execution>
+					<execution>
+						<id>verify</id>
+						<goals>
+							<goal>verify</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
+
+			<plugin>
+				<artifactId>maven-eclipse-plugin</artifactId>
+				<version>2.9</version>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-resources-plugin</artifactId>
+				<version>2.5</version>
+				<configuration>
+					<encoding>UTF-8</encoding>
+				</configuration>
+			</plugin>
+
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-deploy-plugin</artifactId>
+				<version>2.6</version>
+			</plugin>
+
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-gpg-plugin</artifactId>
+				<version>1.5</version>
+				<executions>
+					<execution>
+						<id>sign-artifacts</id>
+						<phase>verify</phase>
+						<goals>
+							<goal>sign</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
+			
```

Comment:
Added GPG plugin too, since I guess it will be needed. But I wonder if its better to use sign-artifacts in another phase except verify since we can't verify to test out branches due to this issue. What do you think?